Add crux links and tags to posts that became crux functions

danielcompton opened the pull request:
A bunch of functions which started life on emacsredux + Prelude have migrated to https://www.github.com/bbatsov/crux. This commit adds links to crux from those blog posts, and tags the posts. Some of the footer mentions were a little bit awkward to include Prelude and Crux. I can reword them if you'd like.

Relates to https://github.com/bbatsov/crux/pull/64.

bbatsov commented:
> I can reword them if you'd like.

Yeah, I think we should reword those. The commands no longer exist directly in Prelude - it simply bundles the `crux` package. Some wording along those lines would be better.

danielcompton commented:
How does this sound?

> This command is available in [crux](https://github.com/bbatsov/crux), named
> `crux-open-with`. This command is also available in
> [prelude](https://github.com/bbatsov/prelude) via the crux package.

bbatsov commented:
Sounds great!

danielcompton committed:
Add crux links and tags to posts that became crux functions

A bunch of functions which started life on emacsredux + Prelude have
migrated to https://www.github.com/bbatsov/crux. This commit adds links
to crux from those blog posts, and tags the posts.

danielcompton commented:
I've made those changes, this is ready for re-review.

bbatsov commented:
Thanks!
